connection/test-utils/src/blockchain.rs

```diff
@@ -118,7 +127,7 @@ impl<L: Ledger + Sync> BlockchainConnection for MockBlockchainConnection<L> {
     fn fetch_block_info(&mut self) -> ConnectionResult<BlockInfo> {
         Ok(BlockInfo {
             block_index: self.ledger.num_blocks().unwrap() - 1,
-            minimum_fees: FeeMap::default_map(),
+            minimum_fees: self.fee_map.as_ref().clone(),
```

Comment:
`as_ref` seems unnecessary
```suggestion
            minimum_fees: self.fee_map.clone(),
```

Reply:
rust amirite?

Reply:
actually, it is necessary to do that. as_ref() turns it into a &BTreeMap